hackclub.com.yaml

```
@@ -2827,3 +2827,8 @@ syria:
   - ttl: 600
     type: CNAME
     value: hackclub-syria.github.io.
+
+pa:
+  - ttl: 1
```

ttl should be >= 600
```suggestion
  - ttl: 600
```